docs/source/guide/dataset_create.md

```diff
@@ -20,26 +20,217 @@ hide_sidebar: true
+
+1. From Label Studio, navigate to the Datasets page and click **Create Dataset**. 
+
+    ![Create a dataset action](/images/data_discovery/dataset_create.png)
+
+2. Complete the following fields and then click **Next**:
+
+    <div class="noheader rowheader">
+
+    | | |
+    | --- | --- |
+    | Name | Enter a name for the dataset. |
+    | Description | Enter a brief description for the dataset.  |
+    | Source | Select AWS S3. |
+
+    </div>
+
+3. Complete the following fields: 
+
+    <div class="noheader rowheader">
+
+    | | |
+    | --- | --- |
+    | Bucket Name | Enter the name of the AWS S3 bucket. |
+    | Bucket Prefix | Enter the folder name within the bucket that you would like to use.  For example, `data-set-1` or `data-set-1/subfolder-2`.  |
+    | File Name Filter | Use glob format to filter which file types to sync. For example, to sync all JPG files, enter `*jpg`. To sync all JPG and PNG files, enter `**/**+(jpg\|png)`.<br><br>At this time, we support the following file types: .jpg, .jpeg, .png, .txt, .text |
+    | Region Name | By default, the region is `us-east-1`. If your bucket is located in a different region, enter it here.  |
+    | S3 Endpoint | Enter an S3 endpoint if you want to override the URL created by S3 to access your bucket. |
+    | Access Key ID | Enter the ID for the access key you created in AWS. Ensure this access key has read permissions for the S3 bucket you are targeting (see "Create an AWS access key" above). |
+    | Secret Access Key | Enter the secret portion of the access key you created earlier. |
+    | Session Token | If you are using a session token as part of your authorization (for example, MFA), enter it here. |
+    | Treat every bucket object as a source file | **Enabled** - Each bucket object will be imported as a separate piece of data (unstructured data). This creates a URL for each bucket object.<br>You should leave this option enabled if your bucket contains BLOB storage files such as JPG, MP3, or similar file types. <br><br>**Disabled** - Disable this option if you are using structured data, such as JSON or CSV files.<br><br>**NOTE:** At this time, we only support unstructured data. |
+    | Recursive scan | Perform recursive scans over the bucket contents if you have nested folders in your S3 bucket. |
+    | Use pre-signed URLs | If your tasks contain `s3://…` links, they must be pre-signed in order to be displayed in the browser. |
+    | Pre-signed URL counter | Adjust the counter for how many minutes the pre-signed URLs are valid. |
+
+    </div>
+
+4. Click **Check Connection** to verify your credentials. If your connection is valid, click **Next**. 
+
+    ![Check Dataset connection](/images/data_discovery/dataset_check_connection_aws.png)
+
+5. Provide a name for your dataset column and select a type. The data type you select instructs Label Studio on how to index your data in a vector database, so that it can be searched using an AI-powered semantic search.
```

Comment:
I want to smooth out some technical details, lmk if you think this is clearer:
> Provide a name for your dataset column and select a data type. This data type tells Label Studio how to store your data in a way that can be searched using its AI-powered semantic search.

Reply:
Rewrote as "Provide a name for your dataset column and select a data type. The data type that you select tells Label Studio how to store your data in a way that can be searched using an AI-powered semantic search."

Also update under the GCS section.